samples/image-metadata/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "image-metadata",
+  "version": "0.1.0",
+  "private": true,
+  "devDependencies": {
+    "@contentful/contentful-extension-scripts": "0.14.0",
+    "@contentful/eslint-config-extension": "0.2.0",
+    "@babel/core": "7.3.4",
+    "@babel/plugin-proposal-class-properties": "7.3.4",
+    "@babel/plugin-transform-runtime": "7.3.4",
+    "@babel/preset-env": "7.3.4",
+    "@babel/preset-react": "7.0.0",
+    "@testing-library/react": "8.0.4",
+    "cssnano": "4.1.10",
+    "contentful-cli": "0.33.2",
+    "eslint": "^6.0.1"
+  },
+  "scripts": {
+    "start": "contentful-extension-scripts start",
+    "build": "contentful-extension-scripts build",
+    "lint": "eslint ./ --ext .js,.jsx,.ts,.tsx",
+    "test": "contentful-extension-scripts test --env=jsdom --watch",
+    "test:coverage": "contentful-extension-scripts test --env=jsdom --coverage",
+    "deploy": "npm run build && contentful extension update --force",
+    "configure": "contentful space use && contentful space environment use",
+    "login": "contentful login",
+    "logout": "contentful logout",
+    "help": "contentful-extension-scripts help"
+  },
+  "dependencies": {
+    "@contentful/forma-36-fcss": "^0.0.20",
+    "@contentful/forma-36-react-components": "^3.11.3",
```

Comment:
Would be cool to update all `dependencies` to latests ones.

Reply:
Done